diff --git a/node_modules/skia-canvas/lib/index.js b/node_modules/skia-canvas/lib/index.js
index 8262aef..f3b0314 100644
--- a/node_modules/skia-canvas/lib/index.js
+++ b/node_modules/skia-canvas/lib/index.js
@@ -83,7 +83,7 @@ const toString = val => typeof val=='string' ? val : new String(val).toString()
 function toSkMatrix(jsMatrix){
   if (Array.isArray(jsMatrix) && jsMatrix.length==6){
     var [a, b, c, d, e, f, m14, m24, m44] = jsMatrix.concat(0, 0, 1)
-  }else if (jsMatrix instanceof geometry.DOMMatrix){
+  }else if (jsMatrix instanceof DOMMatrix){
     var {a, b, c, d, e, f, m14, m24, m44} = jsMatrix
   }
   return [a, c, e, b, d, f, m14, m24, m44]
@@ -91,7 +91,7 @@ function toSkMatrix(jsMatrix){
 
 function fromSkMatrix(skMatrix){
   let [a, b, c, d, e, f, p0, p1, p2] = skMatrix
-  return new geometry.DOMMatrix([
+  return new DOMMatrix([
     a, d, 0, p0,
     b, e, 0, p1,
     0, 0, 1, 0,
